wellcomeml/viz/visualize_clusters.py

```diff
@@ -1,9 +1,12 @@
+import pandas as pd
 from bokeh.plotting import figure, output_file, show
 from bokeh.io import output_notebook, reset_output
-from wellcomeml.viz.palettes import Wellcome33, WellcomeBackground
+from bokeh.models import ColumnDataSource
+from wellcomeml.viz.palettes import (Wellcome33,
+                                     WellcomeBackground, WellcomeNoData)
 
 
-def visualize_clusters(reduced_points: list, radius: float,
+def visualize_clusters(reduced_points: list, clustering, radius: float,
```

Comment:
Now that you're passing `clustering`, you can actually remove this `reduced_points` parameters. Then in the body of the function you can do:

`reduced_points = clustering.reduced_points`